### DISCUSSION TITLE
License Question

### DISCUSSION THREAD

**Discussion by Rdna123** [slint-ui/slint#312] [Q&A] [answered]
I wanting to know if I could still sell a application using the free license by giving access to source code to a customer after they buy it?

**Comment by tronical** [comment, marked as answer]
By using the GPLv3 licensed version of SixtyFPS, the application that you distribute to your customer is a derived work and also covered by the GPLv3. It's perfectly fine to sell that (see also https://www.gnu.org/licenses/gpl-faq.en.html#DoesTheGPLAllowMoney), but note that your customer is allowed to pass copies of your application to others for free, under the terms of the GPLv3.
